bundles/WebToPrintBundle/src/Processor/Chromium.php

```diff
@@ -0,0 +1,189 @@
+<?php
+declare(strict_types=1);
+
+/**
+ * Pimcore
+ *
+ * This source file is available under two different licenses:
+ * - GNU General Public License version 3 (GPLv3)
+ * - Pimcore Commercial License (PCL)
+ * Full copyright and license information is available in
+ * LICENSE.md which is distributed with this source code.
+ *
+ *  @copyright  Copyright (c) Pimcore GmbH (http://www.pimcore.org)
+ *  @license    http://www.pimcore.org/license     GPLv3 and PCL
+ */
+
+namespace Pimcore\Bundle\WebToPrintBundle\Processor;
+
+use Pimcore\Bundle\WebToPrintBundle\Config;
+use Pimcore\Event\DocumentEvents;
+use Pimcore\Bundle\WebToPrintBundle\Event\Model\PrintConfigEvent;
+use Pimcore\Bundle\WebToPrintBundle\Processor;
+use Pimcore\Logger;
+use Pimcore\Bundle\WebToPrintBundle\Model\Document\PrintAbstract;
+use Gotenberg\Gotenberg;
+use Gotenberg\Stream;
+
+class Chromium extends Processor
```

Comment:
The processor should be called `Gotenberg` to be more specific, even if it uses Chromium in the background, the API comes from Gotenberg.

This naming needs then to be changed in other places as well, like the config, ...